src/pages/Game.tsx

```
@@ -0,0 +1,530 @@
+        role={isSelectable ? 'button' : undefined}
+        tabIndex={isSelectable ? 0 : undefined}
+        aria-selected={isSelectable ? isSelected : undefined}
+        onClick={isSelectable ? () => handleCardClick(cardName) : undefined}
+        onKeyDown={isSelectable ? (e: React.KeyboardEvent) => {
+          if (e.key === 'Enter' || e.key === ' ') {
+            e.preventDefault()
+            handleCardClick(cardName)
+          }
+        } : undefined}
+      >
+        <div className="card-name">{cardName}</div>
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)